questionnaire/questionnaire.test.js

```diff
@@ -556,3 +584,361 @@ describe('/questionnaires/{questionnaireId}/sections/answers', () => {
         });
     });
 });
+describe('/questionnaires/{questionnaireId}/submissions', () => {
```

Comment:
As we have a number of top level 'describe' blocks now, it would be nice to wrap them all inside a 'describe' block so that tests can be executed in one go and output read at a glance. This also makes it is more consistent with the rest of our test files.